pkgs/applications/misc/keepassx/community.nix

```
@@ -52,17 +54,21 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./darwin.patch
+    ./quazip5.patch
   ];
 
   cmakeFlags = [
     "-DKEEPASSXC_BUILD_TYPE=Release"
     "-DWITH_GUI_TESTS=ON"
     "-DWITH_XC_AUTOTYPE=ON"
+    "-DWITH_XC_UPDATECHECK=OFF"
     "-DWITH_XC_YUBIKEY=ON"
-    "-DWITH_XC_KEESHARE=ON"
   ]
   ++ (optional withKeePassBrowser "-DWITH_XC_BROWSER=ON")
-  ++ (optional withKeePassHTTP "-DWITH_XC_HTTP=ON")
+  ++ (optional withKeePassKeeShare "-DWITH_XC_KEESHARE=ON")
```

Consider using `optionals`?

```suggestion
  ++ (optionals withKeePassKeeShare [
    "-DWITH_XC_KEESHARE=ON"
    "-DQUAZIP_INCLUDE_DIR=${quazip}/include/quazip"
    "-DQUAZIP_LIBRARIES=${quazip}/lib/libquazip.so"
  ])
```

---

Updated this below with `withKeePassShareSecure` where it should have been originally.